docs/feature_sticky_keys.md

```diff
@@ -0,0 +1,17 @@
+# Sticky Keys
+
+This feature emulates the Sticky Keys accessibility feature found on Windows, macOS and Linux.
+
+Description, from [Wikipedia](https://en.wikipedia.org/wiki/Sticky_keys):
+
+> Sticky keys is an accessibility feature of some graphical user interfaces to assist users who have physical disabilities or help users reduce repetitive strain injury (or a syndrome called the Emacs Pinky). It serializes keystrokes instead of pressing multiple keys at a time, allowing the user to press and release a modifier key, such as Shift, Ctrl, Alt, or the Windows key, and have it remain active until any other key is pressed.
+
+## Usage
+
+First, enable Sticky Keys by setting `STICKY_KEYS_ENABLE = yes` in your `rules.mk`.
+
+Tap (either) `SHIFT` five times to toggle it on or off.
+
+Once toggled on, tap modifiers to have them apply to your next key, i.e. `SHIFT`, `CTRL`, `ALT`, `C` will send `SHIFT` + `CTRL` + `ALT` + `X`.
```

Comment:
You have `C` on one side of the explanation and `X` on the other side, which if I understand this function correctly, would be a typo.

Reply:
I had assumed it was a typo, yeah.   Was more concerned about functionality, and assumed it was an offset error.

Reply:
Yep, typo, will fix.